conf/default-vars.yaml

```diff
@@ -48,7 +48,10 @@ vars:
 
   border:
     size: 1px
-    radius: 3px
+    radius:
+      default: 3px
+      lg: 5px
```

Comment:
it seems odd to me to require modules with a border to have the same exact border radius as buttons. Just seems like unnecessary coupling between the two.  could be swayed though